ui/app/store/actions.js

```diff
@@ -1048,9 +1048,9 @@ export function cancelTypedMsg(msgData) {
   }
 }
 
-export function cancelTx(txData) {
+export function cancelTx(txData, dontShowLoadingIndicator) {
```

Comment:
Why was it necessary to skip the loading indicator in this case?

At the very least we should try to avoid the double-negative case if we're going to add this (e.g. `showLoadingIndicator` rather than `dontShowLoadingIndicator`); `!dontShowLoadingIndicator` is a bit painful to read.

Reply:
This was just to be consistent with the design requests in other parts of swaps, where it was asked that we don't show the orange spinner. `updateTransaction` (on line 834 of this file) was updated in the same way.

Reply:
but yes, I can change the name to `showLoadingIndicator`